src/uint256.h

```diff
@@ -21,72 +22,56 @@ class base_blob
 {
 protected:
     static constexpr int WIDTH = BITS / 8;
-    uint8_t m_data[WIDTH];
+    std::array<uint8_t, WIDTH> m_data;
+    static_assert(WIDTH == sizeof(m_data), "Sanity check");
```

Comment:
[`clang-format`](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy) insists on:
```suggestion
    static_assert(WIDTH == sizeof(m_data), "Sanity check");

```